### PR TITLE
Add documentation about the `codeQL.queryHistory.ttl` setting

### PR DESCRIPTION
**Pull request by aeisenberg**

### CODE REVIEW

**Review by jf205 — DISMISSED** [review]
This change looks good for now, thanks.

Something we should link about at a later date is whether 'Query History: TTL' is an obvious name for this setting. I wouldn't be able to work out what it means without referring to this article. (e.g. what does TTL stand for?)

**Comment by felicitymay** [comment]
@aeisenberg - thanks for opening this PR to fix https://github.com/github/docs-content/issues/8581.

When you're ready for a review by a writer, please add the `ready-for-doc-review` label, so we can put it onto our review board.

**Comment by felicitymay** [comment]
@aeisenberg - thanks for opening this PR.

When you're ready for a review by a writer, please add the ready-for-doc-review label, so we can put it onto our review board.

**Comment by aeisenberg** [comment]
Hi, @felicitymay I am not sure if something got mixed up with the automation, but it looks like this issue has been archived in the Docs content review board. See https://github.com/orgs/github/projects/1367#card-86583559.

Is this PR still on the backlog to be reviewed?

**Comment by felicitymay** [comment]
Hi @aeisenberg - apologies for the delay in reviewing your pull request. It's just been a super-busy time as we get ready for universe.

When you add the `Ready for doc review` label, this adds the PR to our first responder board. The first responder adds it to our docs content review board and archives it. Probably we should update the automation so that the PR goes straight to the review board.

File: docs/codeql/codeql-for-visual-studio-code/customizing-settings.rst [hunk 1]
```diff
@@ -47,6 +47,12 @@ The query history **Format** setting controls how the extension lists queries in
 
 To override the default label, you can specify a different format for the query history items.
 
+
+Changing the retention period for query history items
+-----------------------------------------------------
+
+By default, items in the query history view are retained for 30 days. This value is customizable by changing the "Code QL > Query History: TTL" setting. To retain items indefinitely, set the value to 0.
```
Comment (felicitymay) [inline docs/codeql/codeql-for-visual-studio-code/customizing-settings.rst:54]: Suggest that we make this a little more active and introduce the "TTL" acronym:

```suggestion
By default, items in the query history view are retained for 30 days. You can set a different time to live (TTL) by changing the "Code QL > Query History: TTL" setting. To retain items indefinitely, set the value to 0.
```

**Review by felicitymay — APPROVED** [review]
One small recommendation. Otherwise this looks good to me. Thanks for updating the docs.

Now that the branch for the next release of the CLI has been created, I'm wondering if we might want to merge this into the `codeqlcli-2.11.3` branch instead of `main`?

**Comment by aeisenberg** [comment]
Thanks. That's a good idea. I will retarget on the release branch.

**Review by felicitymay — DISMISSED** [review]

**Comment by aeisenberg** [comment]
@hvitved I believe you are the release manager for codeql 2.11.3. Is it ok to merge this small documentation change into the release?

**Review by hvitved — APPROVED** [review]

**Pull request merged**